src/composables/useBcrosNavigate.ts

```diff
@@ -66,6 +66,9 @@ export const useBcrosNavigate = () => {
   function goToFilingsUI (path: string, params?: { [key: string]: string }) {
     redirect(config.public.filingsURL + path, params)
   }
+  function goToPersonRolesUI (path: string, params?: { [key: string]: string }) {
+    redirect(config.public.peopleRolesURL + path, params)
```

Comment:
Here you reference the config value `peopleRolesURL` but you declared that as `peoplerolesURL` (different casing). Does it work? Did you test it?